docs/DeveloperGuide.adoc

```diff
@@ -677,26 +677,26 @@ Please refer to the User Guide for the full list of commands under both categori
 
 To keep track of navigation state, a boolean `inFolder` is maintained by the `ModelManager`. Other components may retrieve the current state with `ModelManager#isInFolder()`. This is also how the `Command` objects determines whether the command is executable in the present navigation state.
 
-===== Change Command
+===== Change Directory Command
 
 Folder navigation is achieved by the user through the use of the `change` command. As navigating folders do not actually modify folders and their cards, folder navigation does not involve the Storage Component.
 
-The change command has the following formats:
+The change directory command has the following formats:
 
-1. `change ..` - Returns the user to the home directory. This command can only be executed when the user is inside a folder.
-2. `change FOLDER_INDEX` - Enters the folder specified by `FOLDER_INDEX`. This command can only be executed from the home directory, when the user is not in any folder.
+1. `cd ..` - Returns the user to the home directory. This command can only be executed when the user is inside a folder.
+2. `cdFOLDER_INDEX` - Enters the folder specified by `FOLDER_INDEX`. This command can only be executed from the home directory, when the user is not in any folder.
```

Comment:
I think you miss out a space in between cd and FOLDER_INDEX ><

Reply:
Good catch! Thank you :)